plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/support/SupportZip.js

```diff
@@ -125,8 +125,12 @@ Ext.define('NX.coreui.view.support.SupportZip', {
           text: NX.I18n.get('Support_SupportZip_Create_Button'),
           formBind: true,
           glyph: 'xf019@FontAwesome' /* fa-download */,
-          action: 'save',
+          action: 'add',
           ui: 'nx-primary'
+        },
+        {
+          text: NX.I18n.get('SettingsForm_Discard_Button'),
+          action: 'discard'
```

Comment:
I don't think this adds value on this form, more likely to be confusing since it's not clear what you're 'Discard'ing. Also, on a personal note I don't like that choice of wording and we might consider syncing up with CLM on this one - they use 'Cancel' for the same purpose.